procedures/accounts_setup.md

```diff
@@ -0,0 +1,245 @@
+You'll need a few tweaks to your account to get access to everything releng-related:
+* Add privileges for bugzilla group releng confidential (Can be done by manager or bugzilla admin)
+* Add your comms nickname & ldap username as "aliases" for your account
+* log into bugzilla & follow links "Preferences" -> "Account Information"
+* append the aliases, with a leading ':' and enclosed in brackets ('[]') to the "Real Name" field
+* e.g.: "John Doe [:jdoe]"
+
+The product to use is "Release Engineering." There are multiple possible components under that product, so take your best guess or ask for guidance from the team.
+TODO - to revisit once we reorg Bugzilla in November 2022.
+
+:warning:
+    Please speak with your manager to be added to the [releng-security](https://bugzilla.mozilla.org/page.cgi?id=group_admins.html) confidential group.
+
+## Mana
+
+Some internal Mozilla systems (IT, HR) are documented on [mana](https://mana.mozilla.org/). That is behind SSO and you will be granted access to that as soon as LDAP is activated.
+
+## Future access as you need it
+* You will need acccess to our GCP infrastructure. Talk to your manage to be added to the team-releng@firefox.gcp.mozilla.com. Use a ticket similar to [this](https://mozilla-hub.atlassian.net/browse/SVCSE-136) in CloudOps.
+* Ship-it - follow procedure in [here](https://moz-releng-docs.readthedocs.io/en/latest/procedures/release-duty/index.html#how-to-get-ship-it-access) to get access.
+* Balrog - follow procedure in [here](https://moz-releng-docs.readthedocs.io/en/latest/procedures/release-duty/index.html#how-to-get-balrog-access) to get access. A similar related bug was https://bugzilla.mozilla.org/show_bug.cgi?id=1727341.
+* CultureAmp - talk to your manager to set your goals and run the logistics in there for the 1x1 tracking. You'll also need to be added to the RelEng team goals.
+* JIRA - access to JIRA is granted upon SSO access so please make sure to login via the main dashboard. Ask your manager to add you to the RELENG/FFXP boards.
+* Sentry logs - File a bug similar to [this one](https://bugzilla.mozilla.org/show_bug.cgi?id=1731311) to get access to debug various logs in Balrog and more.
+* CloudOps Jenkins - File a bug similar to [Bug 1721444](https://bugzilla.mozilla.org/show_bug.cgi?id=1721444) and talk to your manager to get access to Janekins CloudOps to be able to debug. You should already have access to cloudops-infra repo if you've done the Github section above.
+Once it's properly configured, you'll be able to access it by making some tweaks documented [here](https://github.com/mozilla-services/cloudops-deployment#accessing-jenkins).
+
+### LDAP groups
+* RelEng LDAP flags to access various infrastructure pieces. File a bug similar to [this](https://bugzilla.mozilla.org/show_bug.cgi?id=1732846) and ask your manager for vouching. The flags needed are listed below:
```

Comment:
@gbrownmozilla I used your bug for reference. I'm curious, did you need any other flags to be enabled in the past three months?

Reply:
Oh, yes, actually! I needed "vpn_cloudops_shipit" (bug 1727334).

Reply:
Great catch, added.